src/library/Server/Manager/Directadmin.php

```diff
@@ -424,43 +521,56 @@ private function _request($command, $fields = array(), $post = true)
             $data = $request->getContent();
         } catch (\Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface |
                  \Symfony\Contracts\HttpClient\Exception\HttpExceptionInterface $error) {
-            $e = new Server_Exception('HttpClientException: :error', [':error' => $error->getMessage()]);
-            $this->getLog()->err($e);
-            throw $e;
+            $exception = new Server_Exception('HttpClientException: :error', [':error' => $error->getMessage()]);
+            $this->getLog()->err($exception);
+            throw $exception;
         }
 
-        if(strlen(strstr($data, 'DirectAdmin Login')) > 0) {
+        if(strlen(strstr($data, '<!doctype html>')) > 0 || strlen(strstr($data, 'DirectAdmin Login')) > 0) {
```

Comment:
I think we probably should have a comment here explaining why this logic is used.
It's not particularly clear

Reply:
Ah, I did add a comment somewhere...it might've ended up in my other PR. Essentially, as we're not currently using JSON returns for the DA API, it returns the login page in HTML if certain requests fail. As such, looking for `<!doctype html>` handles this edge case. I'll add a comment above this line!